packages/cli/codemods/__tests_/purge-plugin-legacy-code.test.js

```diff
@@ -0,0 +1,53 @@
+const defineInlineTest = require('jscodeshift/dist/testUtils').defineInlineTest
+const transform = require('../purge-plugin-legacy-code.js')
+const transformOptions = {}
+
+defineInlineTest(
+    transform,
+    transformOptions,
+    "const plugins = require('@dhis2/cli-utils-cypress/plugins')",
+    '',
+    'Removes the legacy import'
+)
+
+defineInlineTest(
+    transform,
+    transformOptions,
+    'plugins(on, config)',
+    '',
+    'Removes the legacy function call'
+)
+
+defineInlineTest(
```

Comment:
Maybe for readability using variables for the input and output would improve the formatting. Looks kind of off with the template string not matching the surrounding indent.